tests/app/phase-team-response/issues-pending/issues-pending.component.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { IssuesPendingComponent } from '../../../../src/app/phase-team-response/issues-pending/issues-pending.component';
+import { Issue, STATUS } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../constants/githubissue.constants';
+import { Team } from '../../../../src/app/core/models/team.model';
+import { IssueService } from '../../../../src/app/core/services/issue.service';
+import { UserService } from '../../../../src/app/core/services/user.service';
+import { USER_Q } from '../../../constants/data.constants';
+
+describe('IssuesPendingComponent', () => {
+    describe('.ngOnInit', () => {
```

Comment:
```suggestion
    describe('.ngOnInit()', () => {
```

This fits our test conventions better

Reply:
Alright!